.gitignore

```diff
@@ -8,3 +8,4 @@
 /spec/reports/
 /tmp/
 .env
+.idea/
```

Comment:
add editor files to global gitignore 
`git config --global core.excludesfile '~/.gitignore'`
`~/.gitignore`

Reply:
@btolarz fixed